drizzlepac/devutils/make_mask_file.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+from astropy.io import fits as pyfits
+import numpy as np
+import pdb
+
+def make_mask_file_old(in_imgname):
+    from scipy import ndimage
+    out_imgname = in_imgname.replace("wht","msk")
+
+    in_wht_hdulist = fits.open(in_imgname)
+    in_sat_hdulist = fits.open(in_imgname.replace("wht","sat"))
+
+    out_hdulist = in_wht_hdulist.copy()
+    sat_data = in_sat_hdulist[0].data
+    ndimage.binary_dilation(sat_data,iterations=11)
+    out_hdulist[0].data = np.where(in_wht_hdulist[0].data+sat_data == 0, 1, 0)
+
+
+    out_hdulist.writeto(out_imgname)
+
+def make_mask_file(drz_image,input_list):
+    from drizzlepac.hlautils import cell_utils as cutils
+    from stwcs.wcsutil import HSTWCS
+
+    s = cutils.SkyFootprint(HSTWCS(drz_image, ext = 1))
+    s.build(input_list)
+    hdu = pyfits.PrimaryHDU(s.total_mask)
+    hdu.data = numpy.where(hdu.data  == 0, 1, 0)
+
+    hdul = pyfits.HDUList([hdu])
+
+    maskfilename = drz_image.replace(drz_image[-9:],"_msk.fits")
+    hdul.writeto(maskfilename)
+    hdul.close()
+    print("Wrote {}".format(maskfilename))
+
+def make_mask_file_actual(imfile):
+    import scipy.ndimage
```

Comment:
make_mask_file.py can actually be deleted. The needed functionality was absorbed into the hla_flag_filter.make_mask_file() subroutine. 
That fact aside, I'm going to continue to use hla_flag_filter.make_mask_file() (at least for now) because the code in the subroutine was the exact same code used to generate the mask during HLA Classic runs, and I want to stay as true to that as possible.

Reply:
drizzlepac/devutils/make_mask_file.py deleted in commit [a169a81](https://github.com/spacetelescope/drizzlepac/pull/462/commits/a169a815a06abc28f3c16bcce03de4855688e226)